services/movie_session.py

```diff
@@ -0,0 +1,63 @@
+import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime.datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+
+    get_movie_id = Movie.objects.get(id=movie_id)
+    get_cinema_id = CinemaHall.objects.get(id=cinema_hall_id)
+    movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=get_cinema_id,
+        movie=get_movie_id
+    )
+    return movie_session
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet[MovieSession]:
+    all_movies = MovieSession.objects.all()
+
+    if session_date is not None:
+        filtered_movies = MovieSession.objects.filter(
+            show_time__date=session_date
+        )
+        return filtered_movies
+    return all_movies
```

Comment:
This is not good better to do as:
```suggestion
    if session_date is not None:
        all_movies = all_movies.filter(
            show_time__date=session_date
        )
    return all_movies
```

Reply:
Because if you will here so much filters it will be so strange